examples/preprocessing/plot_pace.py

```diff
@@ -0,0 +1,372 @@
+"""
+Functional Principal Component Analysis through Conditional Expectation
+=======================================================================
+
+Explores an alternative way to do functional principal component analysis for
+irregularly sampled data.
+"""
+
+# Author: Alejandro Arias Gomez
+# License: MIT
+
+# sphinx_gallery_thumbnail_number = 6
+
+# %%
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.utils import Bunch
+
+from skfda.datasets._real_datasets import fetch_growth
```

Comment:
Do *not* import from private modules. These are just for organizing code internally.